Rakefile

```diff
@@ -37,6 +37,10 @@ if defined?(RSpec)
       t.pattern = "spec/processor_spec.rb"
     end
 
+    RSpec::Core::RakeTask.new(:detect_os) do |t|
+      t.pattern = "spec/helper/detect_os/*_spec.rb"
+    end
+
```

Comment:
I can't do that now because not all tests work. I splitted my big PR to two and when I enable all detect_os tests build will fail. I'll fix it when 614 and 615 PR would be merged.